service_openrc_linux.go

```diff
@@ -77,6 +79,10 @@ func newOpenRCService(i Interface, platform string, c *Config) (Service, error)
 
 var errNoUserServiceOpenRC = errors.New("user services are not supported on OpenRC")
 
+func (s *openrc) ConfigPath() (string, error) {
```

Comment:
As I previously stated, don't simply wrap an un-exported method, rename the existing method.

Reply:
Done